### PR TITLE
Changed some wording

### PR DESCRIPTION
**Pull request by BitesizedLion**
"This allows for better community cooperation and stability" can be reverted if you do not think that it fits.
I also changed some minor things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,26 +7,26 @@ zPractice is a Practice server core for Spigot that aims to deliver the same exp
 
 ## Why use zPractice?
 <br/>
-zPractice is superior in obvious ways like being both free and open-source. This allows for better community interaction and stability over time. As zPractice is open source, we allow for pull requests to be made for the plugin! If you want to contribute to the open source community, feel free to make a pull request to add your wanted features and make your fixes (read on how we manage pull requests)!
+zPractice is superior in obvious ways like being both free and open-source. This allows for better community cooperation and stability over time. As zPractice is open source, we enable for pull requests to be made for the plugin! If you want to contribute to the open-source community, feel free to create a pull request to add your desired features and make your fixes (read on how we handle pull requests)!
```
Comment (ZakShearman) [inline README.md]: "allow" is better than "enable" with the current wording in terms of flow. All of the rest seems good.

File: README.md [hunk 2]
```diff
@@ -7,26 +7,26 @@ zPractice is a Practice server core for Spigot that aims to deliver the same exp
...
 
 ## How can I use zPractice?
 <br/>
-zPractice will be available on Spigot when it is completed, but for the moment, feel free to go ahead and download the source code and import it using Maven. If you don't know how to do this, feel free to search around on Discord for some help, it's simpler than it can be made out to me.
+zPractice will be available on Spigot when it is completed, but for the time being, feel free to go ahead and download the source code and import it using Maven. If you don't know how to do this, feel free to search around on Discord for some help, it's simpler than it can be made out to me.
 
 ## How can I get support for zPractice?
 <br/>
-Support is given over the GitHub issue page (for now) and you can contact me on Discord through my Discord server here: https://discord.gg/fvdXmuS.
+Support is given over the GitHub issue page (for now) and you can contact me on Discord at Zak#6435. More will come in the future.
```
Comment (ZakShearman) [inline README.md]: Could you revert this lol. Most likely happened because you started to edit before I made a change to the page :D.

Comment (BitesizedLion) [reply README.md]: Oopsies will do.

File: README.md (hunk 1 above)
Comment (BitesizedLion) [reply README.md]: Should be fixed :)

File: README.md (hunk 2 above)
Comment (ZakShearman) [reply README.md]: It's still not the Discord link. Check the current README.md.

`Support is given over the GitHub issue page (for now) and you can contact me on Discord through my Discord server here: https://discord.gg/fvdXmuS.`

File: README.md (hunk 1 above)
Comment (ZakShearman) [reply README.md]: Thanks.

**Pull request merged**